mypyc/codegen/emitwrapper.py

```diff
@@ -574,18 +691,30 @@ def generate_wrapper_core(fn: FuncIR, emitter: Emitter,
         emitter.emit_lines('return NULL;')
 
 
-def generate_arg_check(name: str, typ: RType, emitter: Emitter,
-                       error_code: str, optional: bool = False) -> None:
+def generate_arg_check(name: str,
+                       typ: RType,
+                       emitter: Emitter,
+                       error: Optional[ErrorHandler] = None,
+                       *,
+                       optional: bool = False,
+                       raise_exception: bool = True) -> None:
     """Insert a runtime check for argument and unbox if necessary.
 
     The object is named PyObject *obj_{}. This is expected to generate
     a value of name arg_{} (unboxed if necessary). For each primitive a runtime
     check ensures the correct type.
     """
+    error = error or AssignHandler()
     if typ.is_unboxed:
         # Borrow when unboxing to avoid reference count manipulation.
-        emitter.emit_unbox('obj_{}'.format(name), 'arg_{}'.format(name), typ,
-                           error_code, declare_dest=True, borrow=True, optional=optional)
+        emitter.emit_unbox('obj_{}'.format(name),
```

Comment:
I am not asking for a change, but just making a remark: I've really come around to "black-style" for multi-line function calls, where you put the first argument on a new line, indented four spaces